components/common/ConfirmationDialogue.tsx

```diff
@@ -22,7 +22,7 @@ const ConfirmationDialogue = ({ style }: Props) => {
 
   const showActionSheet = () => {
     // To show the Bottom ActionSheet
-    actionSheet.current.show()
+    actionSheet.current!.show()
```

Comment:
```suggestion
    actionSheet.current?.show()
```
Force unwraps cause all sorts of weirdness. (Sorta equivalent to a segfault in C, but JS just gives you undefined instead)